### ISSUE TITLE
Decide on the output format for the annotation tool

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#7]
We want to annotate QPN with this tool (initially by ourselves). In order to do that, the tool needs to output a file format that satisfies these criteria:

- it contains all of the annotations / changes made in the annotation tool (not necessarily the rules, but the outcomes)
- this includes things like labeling columns of assessment tools with a tool name
- the output is easy to submit to the graph as triples (with some conversions)
- doesn't have to be useful or readable for anybody else for now
- doesn't have to be re-insertable into the tool (e.g. to re-run the annotation).

**Comment by jarmoza** [comment]
Next, What is the intended information included in each of the output files?

**Comment by surchs** [comment]
I think what we want for now as a starting output is a subject centric json file. Something like this:
```json
{ 
  "name": "dataset_name_from_filename",
  "type": "dataset",
  "subjects": [
    {
      "id": "sub1",
      "age": 18.1,
      "sex": "male",
      "diagnosis": [
        "SNOMED_term",
        "Other_SNOMED_term"
      ],
      "assessment_tool": [
        "reproschema_tool_name",
        "reproschema_tool_name2"
      ]
    },
    {
      "id": "sub2",
    }
  ]
}
```
Where we have two main entities: datasets (with a name and array of subject) and subjects (with id and all the other good stuff). And then for diagnosis and assessment_tool, having a value inside the arrays would be interpreted as: this is a thing that exists for this subject.

**Comment by surchs** [comment]
Yes, I think for the moment it should. We'll have to rethink this when we think about things like sessions and acquisitions and all that. But right now I would say 1 subject has 1 age and 1 sex.

**Comment by jarmoza** [comment]
Shall we close this issue? Or do you want to keep it around for further revisions to the output format?

**Comment by surchs** [comment]
Yes, I think this is closed now with #163. Hooray :tada:
